<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapshotDialog</class>
 <widget class="QDialog" name="SnapshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snapshot Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Resolutions：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_resolution">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Save Dir：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_dir">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_snap_path">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Format：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_format">
       <item>
        <property name="text">
         <string>JPEG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BMP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RAW</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_period">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Continuous</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Perios(ms)：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_period">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>360000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
